__tests__/seed.test.js

```diff
@@ -57,9 +58,19 @@ describe("GET /api", () => {
         .expect(200)
         .then(({body}) => {
             console.log({body})
-            expect(body.length > 0).toBe(true)
-            expect(body[0])
+            
+                expect.objectContaining({ description: expect.any(String),
+                queries: expect.any(Array),
+                exampleResponse: expect.any(Object)
+                
```

Comment:
You've required in the endpoints above in this file which we could use here to test that the response from this endpoint just matches what we've required in. That way as the file updates this test should always pass.